toolkit/utils/ogmios-client/Cargo.toml

```diff
@@ -15,6 +15,10 @@ serde_json = { workspace = true, features = ["raw_value", "arbitrary_precision"]
 thiserror = { workspace = true }
 time = { workspace = true, features = ["std", "serde", "parsing"] }
 sidechain-domain = { workspace = true, features = ["std"] }
+num-rational = { workspace = true }
+cardano-serialization-lib = { workspace = true }
```

Comment:
I think cardano serialization lib should not leak here. At lease for now we have transformation function in `offchain` crate. We don't pollute ogmios client with CSL.